Element.CloudDistributedLock/CloudDistributedLock.cs

```diff
@@ -12,11 +12,7 @@ public class CloudDistributedLock : IDisposable
         private Timer? timer;
         private bool isDisposed;
 
-
-        public static CloudDistributedLock CreateUnacquiredLock()
-        {
-            return new CloudDistributedLock();
-        }
+        public static readonly CloudDistributedLock UnacquiredLock = new CloudDistributedLock();
```

Comment:
will this work with the `Dispose` pattern? once the first unacquired lock is handed out, if the caller calls `.Dispose()` the static lock will be disposed for subsequent receivers, no?
@AnthonySteele-NewDay